PointOfDispensingApp/config.js

```diff
@@ -4,5 +4,16 @@ export default {
         UNDETERMINED: 0,
         VACCINATION_PROCEED_YES: 1,
         VACCINATION_PROCEED_NO: 2   
-    }	
+    },	
+
+    patientWorkflowState:
+    {
+        INITIAL: 0,
+        RECORD_RETRIEVED: 1,
+        ADMITTED: 2,
+        VACCINATION_COMPLETE: 3,
+        VACCINATION_CANCELED: 4,
+        DISCHARGED: 5,
+        DEMO: 6
```

Comment:
What's `demo` supposed to be?

I'm also a fan of having an `error` or `unknown` state if something unexpected happens

Reply:
Demo:  I didn't end up using this, but I was thinking there might be a need for demo/development mode that makes all left menu tabs accessible at all times.

We should probably add an Error/Unknown state when integrating with the backend APIs - there may be cases where we can't figure out the workflowState based on unexpected data.  For now, I've added it to the enum and created a placeholder action and mutation for it (unknownErrorCondition)

Reply:
Great